estilos/banner.css

```diff
@@ -0,0 +1,181 @@
+* {
+  box-sizing: border-box;
+}
+
+h1 {
+  text-align: center;
+}
+
+body {
+  background-color: orange;
+  padding: 10px;
+  margin: 0;
+}
+
+/*Everything related to this label "a"*/
+a {
+  align-self: center;
+}
+
+a:not(.cards__1 a) {
```

Comment:
It's not wrong, but a better way to add styles is to create a new class and write CSS for it (following the cascade), instead of using negations here.